src/components/ui/DatePicker/DatePicker.svelte

```diff
@@ -268,13 +268,13 @@
         <div>
           <div class="action button" on:mousedown={setToday}>
             <div class="action-icon"><Calendar /></div>
-            <div class="action-label">Today</div>
+            <div class="action-label st-typography-body">Today</div>
           </div>
         </div>
         <div>
           <div class="action button" on:mousedown={clearDate}>
-            <div class="action-icon"><i class="si si-magic_eraser" /></div>
-            <div class="action-label">Clear</div>
+            <div class="action-icon"><i class="bi bi-magic" /></div>
```

Comment:
Can this be kept `si si-magic_eraser`? See https://github.com/NASA-AMMOS/aerie-ui/pull/113

Reply:
Done